### PR TITLE
SET-460 For role selinux, Refactors roles to use vars/ instead of def…

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/SET-460

### CODE REVIEW

File: roles/selinux/defaults/main.yml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 ---
-selinux:
-  state: "{{ override_selinux_state | default('enforcing') }}"
+override_selinux_state: "enforcing"
```
Comment (rpelisse) [inline roles/selinux/defaults/main.yml:2]: @RanabirChakraborty looks, but I would like to take the opportunity to get ride off of the override_ prefix. I would name this vars 'selinux_state' only.

**Comment by RanabirChakraborty** [comment]
@rpelisse please review.

**Review by rpelisse — CHANGES_REQUESTED** [review]
Except for this one change, it looks good. Once you've updated this PR, you can move to the next role (motd and then bashrc)

File: roles/selinux/defaults/main.yml (hunk 1 above)
Comment (RanabirChakraborty) [reply roles/selinux/defaults/main.yml]: Thanks! did the changes. Sure, I'll do the changes in other roles too.

**Pull request merged**